File: app/src/main/java/com/theflexproject/thunder/model/Credits.java
package com.theflexproject.thunder.model;

import java.util.List;

public class Credits {
    private List<Cast> castList;
    private List<Crew> crewList;

    // Constructor
    public Credits(List<Cast> castList, List<Crew> crewList) {
        this.castList = castList;
        this.crewList = crewList;
    }

    // Getters
    public List<Cast> getCastList() {
        return castList;
    }

    public List<Crew> getCrewList() {
        return crewList;
    }
}

File: app/src/main/java/com/theflexproject/thunder/model/Credits.kt
package com.theflexproject.thunder.model

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize
import kotlinx.parcelize.RawValue

@JsonClass(generateAdapter = true)
@Parcelize
data class Credits(
    @Json(name = "cast") val cast: List<@RawValue Cast> = emptyList(),
    @Json(name = "crew") val crew: List<@RawValue Crew> = emptyList()
) : Parcelable
